products.js

```diff
@@ -238,7 +238,7 @@ function tunedPriceFnodes(nodeToCssMap, coeffDollarSign = 4.4, coeffNearDollarSi
     }
 
     function semanticTags(fnode){
-      if (fnode.element.getElementsByTagName('SUP').length > 0 || fnode.element.getElementsByTagName('STRONG') > 0){
+      if (fnode.element.getElementsByTagName('SUP').length > 0){
```

Comment:
Okay! I just removed it to match the effect of the forgotten .length I had before. (adding the .length actually worsened the accuracy)